### ISSUE TITLE
How do you feel?

### ISSUE THREAD

**Issue by githubteacher** [githubschool/animated-invention#169]
🎉  Very Good
👍  Good
😕  Neutral
👎  Bad

**Comment by githubteacher** [comment]
About the basic workflow of Git? (Add, Commit, Push, Status)

**Comment by githubteacher** [comment]
About your conceptual understanding of Remotes, Forks, and Clones?

**Comment by githubteacher** [comment]
About your conceptual understanding of branching and Pull Requests?

**Comment by githubteacher** [comment]
About your conceptual understanding of merging and merge conflicts?

**Comment by RaymondSHANG** [comment]
A little bit fast for me, merge and branch part.

**Comment by githubteacher** [comment]
About git bisect and git revert?

**Comment by githubaf** [comment]
a bit fast

**Comment by ahmi01** [comment]
Bisect was fast. can you review once more?

**Comment by nrprise** [comment]
Would it be best to revert in branch in case issue in revert came up?

**Comment by ahmi01** [comment]
Steps with concept

**Comment by ahmi01** [comment]
Thanks

**Comment by nrprise** [comment]
Can revert result in merge conflict?

**Comment by crichID** [comment]
> Can revert result in merge conflict?

@nrprise yes, a revert can result in a conflict. It isn't technically a `merge conflict` because git isn't doing a merge, but it looks and works the same!

**Comment by ahmi01** [comment]
why your file name is still inde.html

**Comment by nrprise** [comment]
> @nrprise yes, a revert can result in a conflict. It isn't technically a merge conflict because git isn't doing a merge, but it looks and works the same!

@crichID That was actually the root of my original question:

> Would it be best to revert in branch in case issue in revert came up?

I guess is should have said "Would it be best to **do the** revert **action** in **a** branch in case issue in revert came up **(like a merge conflict)**?

**Comment by crichID** [comment]
> why your file name is still inde.html

@ahmi01 the file name is still inde.html because we are actually checking out to a branch that was created before we fixed it. So the branch `shape-colors` was created earlier in history - we are picking it up and starting to work on it, but it is out of date based on the more recent changes we have made. 
@crichID got it. my file name is changed to index.html. thanks.

**Comment by crichID** [comment]
> Would it be best to do the revert action in a branch in case issue in revert came up (like a merge conflict)?

@nrprise yes, you are exactly correct - and I apologize for not reading up further :wink:

**Comment by AFTERWAKE** [comment]
~unrelated to the course~
I've used atom for a while, and I just picked up the atom beta with the better github integration tools. Do you happen to know when that will be coming to the main release?
:+1: thanks!

**Comment by francesco-damore** [comment]
can we use http://git-school.github.io/visualizing-git/ at any time or is just available for today?

**Comment by williamsmi** [comment]
we did the commit to the change-speed branch.  do we need to do push or merge to our gh-pages branch?
